engine/dlib/src/mbedtls/library/net_sockets.c

```diff
@@ -392,9 +398,12 @@ int mbedtls_net_accept( mbedtls_net_context *bind_ctx,
         if( getsockname( client_ctx->fd,
                          (struct sockaddr *) &local_addr, &n ) != 0 ||
             ( bind_ctx->fd = (int) socket( local_addr.ss_family,
-                                           SOCK_DGRAM, IPPROTO_UDP ) ) < 0 ||
-            setsockopt( bind_ctx->fd, SOL_SOCKET, SO_REUSEADDR,
-                        (const char *) &one, sizeof( one ) ) != 0 )
+                                           SOCK_DGRAM, IPPROTO_UDP ) ) < 0
+#if !defined(__EMSCRIPTEN__)
+            || setsockopt( bind_ctx->fd, SOL_SOCKET, SO_REUSEADDR,
+                        (const char *) &one, sizeof( one ) ) != 0
+#endif
```

Comment:
I don't remember if we ever use preprocessor in the middle of expression. It makes it harder to read, let me ask @JCash if it's fine